content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -221,3 +263,21 @@ To provide domain verification:
 If your validator's master private key is compromised, you must revoke it immediately and permanently.
 
 For information about how to revoke a master key pair you generated for your validator using the `validator-keys` tool, see [Key Revocation](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md#key-revocation).
+
+
+
+## Verify Your Connection to Network
+
+Here are some methods you can use to verify that your validator has a healthy connection to the XRP Ledger network:
+
+- Use the [`peers`](peers.html) command to return a list of all `rippled` servers currently connected to your validator. If the `peers` array is `null`, you don’t have a healthy connection to the network. If you've set up your validator using the instructions in this document, the `peers` array should include the same number of objects as the number of peers defined in your `[ips_fixed]` stanza. ***TODO: I have both r.ripple.com 51235 and zaphod.alloy.ee 51235 in my [ips_fixed] - and sometimes I get connect to only 1 peer. Is this a problem, or is it okay? If this is okay, I guess we can't say that the number of peers in ips_fixed and the number of peers returned will always be the same. If you are not using [peer_private] - what is a healthy minimum number of peers?***
+
+    - If you are running an untrusted validator, haven't already set up connections to public hubs, and are having trouble maintaining a reliable and safe connection to the network, see [Connect to the Network Using Public Hubs](#connect-to-the-network-using-public-hubs).
+
+    - If you are running a trusted validator, haven't already set up connections to proxies, and are having trouble maintaining a reliable and safe connection to the network, see [Connect to the Network Using Proxies](#connect-to-the-network-using-proxies).
+
+- Use the [`server_info`](server_info.html) command to return some basic information about your validator. The `server_state` should be set to `proposing`. It may also be set to `full` or `validating`, but only for a few minutes before moving into `proposing`.
+
+    If the `server_state` does not spend the majority of its time set to `proposing`, it may be a sign that your validator is unable to fully participate in the XRP Ledger network. ***TODO: accurate?*** For more information about server states and using the `server_info` endpoint to diagnose issues with your validator, see [`rippled` Server States](rippled-server-states.html) and [Get the `server_info`](diagnosing-problems.html#get-the-server-info).
```

Comment:
Yes, 'proposing' is the healthy state for validators. On initial set up it may take a few minutes for the validator to sync with the network and get there.